src/main/steps/applicant/support-you-need-during-case/SupportYouNeedDuringCaseController.ts

```diff
@@ -85,4 +50,46 @@ export class SupportYouNeedDuringYourCaseController extends PostController<AnyOb
       throw new Error('SafetyConcernsPostController - Case could not be updated.');
     }
   }
+
+  private getSupportYouNeedForApplicant(req: AppRequest<Partial<Case>>) {
+    if ('C100' === req.session.userCase.caseTypeOfApplication) {
+      req.session.userCase?.applicants?.forEach((applicant: Applicant) => {
+        if (applicant?.value?.user?.idamId === req.session?.user.id) {
+          if (req.url.includes('support-you-need-during-case')) {
```

Comment:
Do we really need this url check?

Reply:
We really dont need that check .. removed for FL401 respondent, C100 respondent and also checked the flow .. they are working fine.